.deployment.config.json

```diff
@@ -41,27 +45,31 @@
       "id": "deploy-headless-to-s3-version",
       "s3": {
         "bucket": "{terraform.infra.infra.bucket_binaries}",
-        "directory": "proda/StaticCDN/headless/v$[PRERELEASE]",
+        "directory": "proda/StaticCDN/headless/v$[VERSION]",
         "source": "packages/headless/dist/browser",
         "parameters": {
           "acl": "public-read"
-        },
-        "prd": {
-          "directory": "proda/StaticCDN/headless/v$[VERSION]"
         }
       }
     },
     {
       "id": "deploy-atomic-to-s3-version",
       "s3": {
         "bucket": "{terraform.infra.infra.bucket_binaries}",
-        "directory": "proda/StaticCDN/atomic/v$[PRERELEASE]",
+        "directory": "proda/StaticCDN/atomic/v$[VERSION]",
         "source": "packages/atomic/dist/atomic",
         "parameters": {
           "acl": "public-read"
-        },
-        "prd": {
-          "directory": "proda/StaticCDN/atomic/v$[VERSION]"
+        }
+      }
+    },
+    {
+      "id": "wait-for-dev-deployment-request",
+      "manual": {
```

Comment:
This step waits for approval from the dev team. This is how we'll trigger new releases now. QA approval will still be necessary for prod though.

Reply:
Why is this manual step needed?

Reply:
I removed the manual approval (Deploy button in Slack) that was required to transition to QA in favor of adding one here instead so that we can create a release bump commit in the dev pipeline after approval.